src/styles/common.css

```diff
@@ -16,4 +78,74 @@ top: 20%;
 }
 .genderDesign {
 top: 10rem;
-}
\ No newline at end of file
+}
+.navBarstyle{
+    display: flex;
+    background: #2e4b64;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 60px;
+    padding-bottom: 1.3rem;
+  }
+  .toppaddingstyle {
+    padding-top: 15px;
+    display: flex;
```

Comment:
display: flex; justify-content: center;  align-items: center;
Add these styles as a separate class as used in many places